about/quickstart.md

```diff
@@ -47,13 +47,13 @@ You saw above that access to the diagnostic overview required you to authenticat
 in your brower. From the command line, you can see that 
 
 ```shell
-curl -v 'http://localhost:8080/qotm/quote/5?json=true'
+curl -Lv 'http://localhost:8080/qotm/quote/5?json=true'
```

Comment:
Is "-Lv" a valid argument (as opposed to "-L -v")?

Reply:
Yes, single `-` flags may be combined.

Reply:
Yes, it is.